cmd/minikube/cmd/dashboard.go

```diff
@@ -40,17 +44,34 @@ var dashboardCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		api := libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))
 		defer api.Close()
-		url, err := cluster.GetServiceURL(api, "kube-system", "kubernetes-dashboard")
+		s, err := cluster.GetHostStatus(api)
 		if err != nil {
-			glog.Errorln("Error accessing the kubernetes dashboard (is minikube running?): Error: ", err)
+			glog.Errorln("Error getting machine status:", err)
 			os.Exit(1)
 		}
+		if s != state.Running.String() {
+			fmt.Fprintln(os.Stdout, "minikube is not currently running so no dashboard is available")
```

Comment:
I copied what `minikube status` did as I assumed it was an error if someone tried to open a service when minikube was unable to get a status.  I can remove this, what should happen in this case though?